src/test/java/org/cactoos/http/HtWireTest.java

```diff
@@ -78,7 +80,12 @@ public void worksWithProvidedHostNameAndPort() throws IOException {
                         new JoinedText(
                             "\r\n",
                             "GET / HTTP/1.1",
-                            String.format("Host:%s", home.getHost())
+                            new UncheckedText(
```

Comment:
@alex-semenyuk Why `UncheckedText` here? this test already declares it throws an `IOException`

Reply:
Fixed